airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/manifest.yaml

```diff
@@ -0,0 +1,235 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: [ "items", "data" ]
+  requester:
+    type: HttpRequester
+    url_base: "https://api.getbase.com/v2"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_token'] }}"
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```

Comment:
This is not corresponding to existing connector where you have a specific `next_page_token` function. Please implement correct pagination.